How to ask Stampy for questions.md

```diff
@@ -0,0 +1,75 @@
+# How to ask Stampy for questions
+
+Remember that you need to prefix your message with "stampy, " or "s, ".
```

Comment:
we can also reply to previous messages - do we want to mention that in this file?

question for @robertskmiles - stampy used to treat messages in `#talk-to-stampy` as if everything was addressed directly to him, but that doesn't seem to work recently - did we remove that feature on purpose, or is that a bug and stampy should be more willing to answer in that channel?